src/extension.ts

```diff
@@ -1,27 +1,17 @@
 import * as vscode from 'vscode';
 
 import { setupCompletionItemProvider } from './completion/setupCompletionItemProvider';
-import * as manifest from './manifest';
+import { setupXdlManifest } from './manifest';
 import { setupPreview } from './preview/setupPreview';
 
 export async function activate(context: vscode.ExtensionContext) {
   try {
-    await manifest.activateGlobalSchema(context);
+    await setupXdlManifest(context);
     await setupCompletionItemProvider(context);
     await setupPreview(context);
   } catch (error) {
     vscode.window.showErrorMessage(
-      `Oops, looks like we couldn't activate the Expo manifest tools: ${error.message}`
-    );
-  }
-}
-
-export async function deactivate() {
-  try {
-    await manifest.deactivateGlobalSchema();
-  } catch (error) {
-    vscode.window.showErrorMessage(
-      `Oops, looks like we couldn't deactivate the Expo manifest tools: ${error.message}`
+      `Oops, looks like we can't fully activate Expo Tools: ${error.message}`
```

Comment:
When would this happen? Network error?

Reply:
No, I don't think we download anything manually anymore. It's just whenever one of the vscode APIs throws an error. We could also let vscode just handle it instead of informing the user?

Reply:
It just serves as a general catch-all to let the users know something is not right. It could help surface future issues like: https://github.com/expo/vscode-expo/issues/87